### PR TITLE
TBW-24 Bug fixes and Sticky Services Bar

### PR DESCRIPTION
**Pull request by Liannus**
Changelog:
- Fixed mobile services navbar to stick on scrolling

### CODE REVIEW

File: frontend/src/Pages/ServicesPage/ServicesPage.scss [hunk 1]
```diff
@@ -4,10 +4,27 @@
...
+    margin-top: $default-spacing !important;
     display: flex;
   }
+
+  .sticky-nav {
+    position: -webkit-sticky;
+    position: sticky;
+    margin-left: calc(50% - 50vw) !important;
+    margin-right: calc(50% - 50vw) !important;
+    background-color: white;
```
Comment (ForkBombGIT) [inline frontend/src/Pages/ServicesPage/ServicesPage.scss]: can we use $color-white

File: frontend/src/Components/CustomServicesNav/CustomServicesNav.scss [hunk 2]
```diff
@@ -16,16 +22,20 @@
   color: $color-gray;
   h2 {
     font-size: $font-size-subtitle;
-    margin-bottom: $default-spacing / 2;
+    @include desktop {
+      margin-bottom: $default-spacing / 2;
+    }
     color: black;
```
Comment (ForkBombGIT) [inline frontend/src/Components/CustomServicesNav/CustomServicesNav.scss]: can we change this to $color-black

File: frontend/src/Pages/ServicesPage/ServicesPage.scss (hunk 1 above)
Comment (Liannus) [reply frontend/src/Pages/ServicesPage/ServicesPage.scss]: Yep, done

File: frontend/src/Components/CustomServicesNav/CustomServicesNav.scss (hunk 2 above)
Comment (Liannus) [reply frontend/src/Components/CustomServicesNav/CustomServicesNav.scss]: yep, although used `$color-primary` instead.

**Pull request merged**